tests/MockAPNSServer/package.json

```diff
@@ -1,6 +1,5 @@
 {
   "dependencies": {
-    "prettier": "^2.1.1",
```

Comment:
So I'm guessing Prettier doesn't play nicely with the WordPress.com linter?

Out of scope for this PR, but maybe there's a way to write a config for Prettier that's in line with those standards? Sounds like a good hackweek project.